packages/app/src/app/hooks/useWorkspaceLimits.ts

```diff
@@ -26,41 +54,47 @@ export const useWorkspaceLimits = (): WorkspaceLimitsReturn => {
       authorization === TeamMemberAuthorization.Admin ||
       authorization === TeamMemberAuthorization.Write
   );
+
   const numberOfEditors = isTeamSpace
     ? editorOrAdminAuthorizations?.length || 1
     : 1; // Personal
 
   const hasMaxNumberOfEditors =
-    isFree === true && numberOfEditors === limits.maxEditors;
+    debugLimits?.restrictEditors ||
+    (isFree === true && numberOfEditors === limits.maxEditors);
 
   const numberOfEditorsIsOverTheLimit =
-    isFree === true &&
-    limits.maxEditors !== null &&
-    numberOfEditors > limits.maxEditors;
+    debugLimits?.restrictEditors ||
+    (isFree === true &&
+      limits.maxEditors !== null &&
+      numberOfEditors > limits.maxEditors);
 
   const publicProjectsQuantity = usage.publicProjectsQuantity;
   const maxPublicProjects = limits.maxPublicProjects;
 
   const hasMaxPublicRepositories =
-    isFree === true &&
-    maxPublicProjects !== null &&
-    publicProjectsQuantity >= maxPublicProjects;
+    debugLimits?.hasMaxPublicRepositories ||
+    (isFree === true &&
+      maxPublicProjects !== null &&
+      publicProjectsQuantity >= maxPublicProjects);
 
   const privateRepositoriesQuantity = usage.privateProjectsQuantity;
   const maxPrivateRepositories = limits.maxPrivateProjects;
 
   const hasMaxPrivateRepositories =
-    isFree === true &&
-    maxPrivateRepositories !== null &&
-    privateRepositoriesQuantity > maxPrivateRepositories;
+    debugLimits?.hasMaxPrivateRepositories ||
+    (isFree === true &&
+      maxPrivateRepositories !== null &&
+      privateRepositoriesQuantity > maxPrivateRepositories);
 
   const publicSandboxesQuantity = usage.publicSandboxesQuantity;
   const maxPublicSandboxes = limits.maxPublicSandboxes;
 
   const hasMaxPublicSandboxes =
-    isFree === true &&
-    maxPublicSandboxes !== null &&
-    publicSandboxesQuantity >= maxPublicSandboxes;
+    debugLimits?.hasMaxPrivateRepositories ||
+    (isFree === true &&
+      maxPublicSandboxes !== null &&
+      publicSandboxesQuantity >= maxPublicSandboxes);
```

Comment:
This is something I need to get used to haha, it feels like it is making the code a little bit more convoluted but the Leva usage is just so useful 🤯!

Reply:
@tristandubbeld I agree! Hopefully, the usage will be limited to the hooks and we won't be adding too many controls.